roles/migrationcontroller/tasks/main.yml

```diff
@@ -519,6 +519,7 @@
     - name: "Set up mig controller configmap"
       k8s:
         state: present
+        apply: yes
```

Comment:
I suppose this will remove anything the user has added manually to this configmap, which is probably OK, just want to call it out.

Reply:
Wasn't this a bug before? I feel like at the very basic level this was the bug that I am fixing here.

Reply:
This type thing can be used to hackily workaround some issues until we release an updated version, but yeah I think it wasn't intended.